pkg/runtime/lock.go

```diff
@@ -121,6 +123,14 @@ func (l *filelock) Release() error {
 	return os.Remove(l.path)
 }
 
+// Unlock releases the ownership of a lock.
+// Returns an error if the invoking process is not the current owner
+func (l *filelock) Owner() int {
```

Comment:
Why are we wrapping `getOwner` into `Owner`, where the latter simply discards an error? Perhaps we could just make `getOwner` public and let implementations discard the error if they wish.

Reply:
@pablochacin This has a thumbs up but I don't see any change in the code. Do you want to discuss this further?

Reply:
In https://github.com/grafana/xk6-disruptor/pull/279/commits/3fbfb9411154aae3fae98a676fdc4a6a7841363f i actually removed `getOwner` and use `Owner` as returning and error and an int seemed redundant except for one case that we were not actually handling.

Reply:
Not sure if GitHub interface played a game on me or if I need more coffee, but somehow I still saw `getOwner` in there. Change looks great :)